src/openssl/pkcs5.cr

```diff
@@ -8,4 +8,23 @@ module OpenSSL::PKCS5
     end
     buffer
   end
+
+  def self.pbkdf2_hmac(secret, salt, iterations = 2**16, algorithm : Symbol = :sha1, key_size = 64) : Bytes
```

Comment:
Use `String` instead of `Symbol`. Here, I'd rather use an `enum`

Reply:
Sure!
I declared a new enum named DigestAlgorithm.
I'm not sure about the following:
- Location (should I declare this inside the module?)
- Casing (for acronims in enum members)
- Enum naming (is the name DigestAlgorithm a good name?)